frontend/app/templates/work_packages/tabs/activity.html

```diff
@@ -10,12 +10,14 @@
           <h3 class="activity-date"
               ng-if="currentDate !== (activities[$index-1].props.createdAt | date:'longDate')"
               ng-bind="currentDate"/>
-          <user-activity work-package="workPackage"
-                         activity="activity"
-                         activity-no="activityNo"
-                         is-initial="isInitial"
-                         input-element-id="'add-comment-text'">
-          </user-activity>
+            <activity-entry work-package="workPackage"
+                           activity="activity"
+                           activity-no="activityNo"
+                           is-initial="isInitial"
```

Comment:
You were goddamn right with your fear. I've committed the changeset before the WP was created, and it is indeed the first journal. Should this be addressed in the frontend?

![bildschirmfoto 2015-08-13 um 13 09 10](https://cloud.githubusercontent.com/assets/459462/9248497/88894942-41bc-11e5-89e3-db4b5313865b.png)